pennylane_lightning/lightning_qubit.py

```diff
@@ -116,21 +116,20 @@ def apply(self, operations, rotations=None, **kwargs):
             super().apply(operations, rotations=rotations, **kwargs)
             return
 
-        for i, operation in enumerate(operations):  # State preparation is currently done in Python
+        # State preparation is currently done in Python
```

Comment:
Maybe also assert len(operations) > 1?

Reply:
using a try-catch now to dismiss index errors.  I'm assuming that's what you were getting at?